File: Soal-3/PRAK103-2410817310014-INDRA_SURYADILAGA.py
a=5
b=6
x=7
y=8

charvar = ['a', 'b', 'x', 'y']
datavar = [a, b, x, y]

for char, data in zip(charvar, datavar):
    print(f'Variabel {char} bernilai {data}')

print(f'Jumlah dari a ditambah  b dikali x dibagi y adalah {float(a+b*x/y):.4f}')

File: Soal-3/PRAK103-2410817310014-INDRA_SURYADILAGA.c
#include <stdio.h>

int main() {
    float a = 5;
    float b = 6;
    float x = 7;
    float y = 8;
    
    printf("Variabel a bernilai %.0f\n",a);
    printf("Variabel b bernilai %.0f\n",b);
    printf("Variabel x bernilai %.0f\n",x);
    printf("Variabel y bernilai %.0f\n",y);
    printf("Jumlah a ditambah b dikali x dibagi y adalah %.4f\n", a+b*x/y);
    return 0;
}